### ISSUE TITLE
Duplicates in the AllstarsFull.csv in core for playerid: aparilu01

### ISSUE THREAD

**Issue by McPowerShell** [chadwickbureau/baseballdatabank#145]
| playerID  | yearID | gameNum | gameID       | teamID | lgID | GP | startingPos |
|-----------|--------|---------|--------------|--------|------|----|-------------|
| aparilu01 | 1962   | 0       | ALS196207100 | CHA    | AL   | 1  | 6           |
| aparilu01 | 1962   | 0       | ALS196207100 | CHA    | AL   | 1  | 6           |
| aparilu01 | 1962   | 0       | NLS196207300 | CHA    | AL   | 1  | 6           |
| aparilu01 | 1962   | 0       | NLS196207300 | CHA    | AL   | 1  | 6           |

**Comment by McPowerShell** [comment]
**Luis Aparicio** 
*Luis Aparicio All-Star Fielding Stats*

Todos: 

- Remove the 2 NLS196207300 since he only played for the AL.
- Remove the AL dup of the 1962 ALS196207100

| Year | Team | POS | G | GS | OUTS | TC | TC/G | CH | PO | A | E | DP | PB | CASB | CACS | FLD% | RF |
|------|------|-----|---|----|------|----|------|----|----|---|---|----|----|------|------|------|----|
| 1958 | AL   | SS  | 1 | 1  | 18   | 2  | 2.0  | 2  | 1  | 1 | 0 | 0  | n/a | n/a  | n/a  | 1.000| 3.00|
| 1959 | AL   | SS  | 2 | 2  | 48   | 9  | 4.5  | 9  | 5  | 4 | 0 | 1  | n/a | n/a  | n/a  | 1.000| 5.06|
| 1960 | AL   | SS  | 1 | 0  | 12   | 2  | 2.0  | 2  | 1  | 1 | 0 | 0  | n/a | n/a  | n/a  | 1.000| 4.50|
| 1961 | AL   | SS  | 1 | 1  | 27   | 4  | 4.0  | 4  | 1  | 3 | 0 | 0  | n/a | n/a  | n/a  | 1.000| 4.00|
| 1962 | AL   | SS  | 2 | 2  | 39   | 10 | 5.0  | 10 | 5  | 5 | 0 | 2  | n/a | n/a  | n/a  | 1.000| 6.92|
| 1963 | AL   | SS  | 1 | 0  | 9    | 0  | 0.0  | 0  | 0  | 0 | 0 | 0  | n/a | n/a  | n/a  | .000 | 0.00|
| 1970 | AL   | SS  | 1 | 1  | 33   | 3  | 3.0  | 3  | 0  | 3 | 0 | 0  | n/a | n/a  | n/a  | 1.000| 2.45|
| 1971 | AL   | SS  | 1 | 1  | 27   | 3  | 3.0  | 3  | 1  | 2 | 0 | 0  | n/a | n/a  | n/a  | 1.000| 3.00|

Duplicate Entries: The dataset includes duplicate entries for player aparilu01 (Luis Aparicio) in the 1962 All-Star game(s). Each entry should be unique to a specific game.

Multiple Game IDs for the Same Year: For the year 1962, there are two different game IDs (NLS196207300 and ALS196207100). Typically, each All-Star game in a given year should have a unique game ID. However, it's worth noting that in some years, including 1962, two All-Star games were played. These should be distinguished by different gameNum values (e.g., 0 and 1).

Inconsistent GameNum Values: The gameNum column, which should differentiate between multiple games in a year (if applicable), is set to 0 for all entries. If these entries are indeed for two different games in 1962, the gameNum should reflect this (e.g., 0 for the first game and 1 for the second game).

Game ID Formatting: The format of the game IDs seems inconsistent (NLS196207300 and ALS196207100). Typically, a game ID would follow a consistent format. The prefixes NLS and ALS are unusual. It's more common to see a format like ASG1962, potentially with an additional identifier if there were two games that year.

Redundancy in Information: The data repeats the same information (teamID, lgID, GP, startingPos) which might be unnecessary if it's for the same player in the same game. This could be streamlined unless there's a specific reason for the repetition.

In summary, the main issues are the duplication of entries, potentially incorrect gameNum values, unusual game ID formatting, and possibly unnecessary repetition of information. To resolve these, each game should have a unique identifier, and any duplicate entries should be removed or corrected to accurately reflect different games or roles within the same game.
